### DISCUSSION TITLE
Untargeted metabolomic

### DISCUSSION THREAD

**Discussion by Melogmo** [systemsomicslab/MsdialWorkbench#109] [Show and tell]
Hello Dr. Kozo

**Comment by kozo2** [comment]
Hi Melogmo,
This is not a research opportunity and supervision, but I comment to you as a MS-DIAL community member.

If you are in a hurry to do molecular networking anyway, do
it using MS-DIAL version 4 series.
http://prime.psc.riken.jp/compms/msdial/main.html
https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking-with-ms-dial/

MS-DIAL has version 4 and 5 series, but the 5 series is not yet ready
to make molecular networking available.

**Reply by Melogmo** [reply]
Hi Kozo, 
i hope you are doing well. 
I have few concerns with MS-DIAL. 
I downloaded application but when i tried to open new project to upload data, the application disappear. 
when i click on next, the application disappear
I don't know how should i do. 
Please can you help me?

**Reply by kozo2** [reply]
Hi @Melogmo 

Sorry for my late reply.
Can you record your screen and share it with me?
By the way, you can attach Video (.mp4, .mov, .webm) up to 10MB to this comment section.

If that is difficult, multiple screen shots would be fine.

**Comment by Melogmo** [comment]
Hello Kozo, 
I hope you are doing well. 
I am not hearing from you since last week. 
Please, i am still have worries with MS-DIAL. 
Please, may you found a solution?

**Reply by kozo2** [reply]
Sorry for my late reply.
Can you record your screen and share it with me?
By the way, you can attach Video (.mp4, .mov, .webm) up to 10MB to this comment section.

If that is difficult, multiple screen shots would be fine.

**Comment by Melogmo** [comment]
Hello Kozo,
I succeeded in fixing the problem.
The problem came from dotnet and the language.

But, I have other concerns. I uploaded the data (.mzml) and after
processing the extension of outcome was .dcl and .pai2 and not .mdt like.
These formats are non compatible with MS-DIAL.
I tried to convert the raw data (.pdd) to .abf using ABFconvert but i
received this message:
[image: image.png]

I downloaded and installed MSFileReader but it always no working
[image: image.png]

Please, can you show me the right way that I should follow?

Thank you in advance


Le ven. 23 juin 2023 à 06:09, Kozo Nishida ***@***.***> a
écrit :

> Sorry for my late reply.
> Would it be possible to ask you to share a video recording of your issue?
> That would be easy to do with
>
>    - How to record your screen on Windows 11
>    <https://www.microsoft.com/en-us/windows/learning-center/how-to-record-screen-windows-11>
>    - Attaching files
>    <https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/systemsomicslab/MsdialWorkbench/discussions/109#discussioncomment-6258476>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5L4I5DT4GU4QJODE4JFPX3XMUQJLANCNFSM6AAAAAAZEQWJKE>
> .
> You are receiving this because you were mentioned.Message ID:
> <systemsomicslab/MsdialWorkbench/repo-discussions/109/comments/6258476@
> github.com>
>


-- 

Yanick Kevin Dongmo Melogmo, MSc

PhD student, Biochemistry

Anti-Microbial and Biocontrol Agent Unit

Laboratory for Phytobiochemistry and Medicinal Plants Studies
Faculty of Sciences UYI
Email: ***@***.***

Gmail: ***@***.***

Outlook: ***@***.***

skype: Yanick_MELOGMO

Tel: (Whatsapp):

(+237) 678900517

             693319236

**Reply by MB-UP** [reply]
Hi Melogmo

Can I please ask how did you fix the problem of the MSDIAL software kicking you out (application disappearing). Im currently struggling with that. 

Thanks, 
MB

**Comment by Melogmo** [comment]
Hello
Make sure that your system is in English.


Thanks

**Reply by MB-UP** [reply]
Hi thank you Melogmo

I do use English i even downloaded the new version of MSDIAL but still no luck.

**Comment by Melogmo** [comment]
Hi
Download and install the latest version of Microsoft.net

**Reply by MB-UP** [reply]
I will try that thank you

**Reply by MB-UP** [reply]
I downloaded Microsoft. NET 9 but it didn't assist improve. I'm wondering if it's because I'm using windows 11.

**Comment by JudeLeung0627** [comment]
Hi Melogmo,

Do you know how to use the molecular network in MS DIAL 5. I had a question, Some data were collected by Agilent Qtof 6546, resulting in all the identified compounds with 3 kinds of CEs (10 ev, 20 ev, 40 ev) had 3 mass spectras and even more.  And I do not which CEs' mass spectra the MS Dial choose to molecular network.  So how to deal with these data, and let the molecular network more accurate.

Thanks
